### PR TITLE
optional fields in Device when non strict, svd-parser 0.13.1

### PR DESCRIPTION
**Pull request by burrbull**

### CODE REVIEW

**Comment by adamgreig** [comment]
Will strict mode still check these exist?

**Comment by burrbull** [comment]
> 

Yes.
https://github.com/rust-embedded/svd/blob/23c628f737e95f9adfbfe1e40f36780de31fd485/svd-rs/src/device.rs#L251

**Comment by burrbull** [comment]
Wait for check in https://github.com/rust-embedded/svd2rust/pull/556

**Comment by burrbull** [comment]
Made typo. Sorry

File: svd-parser/CHANGELOG.md [hunk 1]
```diff
@@ -23,7 +27,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 Previous versions in common [changelog][../CHANGELOG.md].
 
-[Unreleased]: https://github.com/rust-embedded/svd/compare/v0.13.0...HEAD
+[Unreleased]: https://github.com/rust-embedded/svd/compare/v0.13.1...HEAD
```
Comment (adamgreig) [inline svd-parser/CHANGELOG.md:30]:
```suggestion
[Unreleased]: https://github.com/rust-embedded/svd/compare/svd-parser-v0.13.1...HEAD
```

**Review by adamgreig — APPROVED** [review]
bors merge

**Pull request merged**